app/src/main/java/eu/kanade/tachiyomi/data/database/DbOpenCallback.kt

```diff
@@ -21,7 +22,7 @@ class DbOpenCallback : SupportSQLiteOpenHelper.Callback(DATABASE_VERSION) {
         const val DATABASE_NAME = "tachiyomi.db"
 
         /** Version of the database. */
-        const val DATABASE_VERSION = 41
+        const val DATABASE_VERSION = 43
```

Comment:
why 43 and not 42?

Reply:
Oh, that was because I needed to test the rename migration, I can bump to 42 instead, I'll just need to clear the data in the emulator